test_setup.js

```diff
@@ -250,6 +250,12 @@ module.exports = async function (done) {
     assertEnvVars(basicVars.concat(googleVars, submitDataVars));
     console.log('TEST_DATA_PATH: ', process.env.TEST_DATA_PATH);
 
+    // If testedEnv is not defined, infer FQDN based on the NAMESPACE
+    if (process.env['testedEnv'] === '' || process.env['testedEnv'] === undefined) {
```

Comment:
i dont think javascript style encourages use of the triple equals against the undefined keyword. i think the string comparison like so is preferred: 
`typeof process.env['testedEnv'] === 'undefined'`

Reply:
source: https://stackoverflow.com/questions/3390396/how-to-check-for-undefined-in-javascript

Reply:
Oh, I was reproducing _prior art_ found on the same script -> Line 17:
```
const inJenkins = (process.env.JENKINS_HOME !== '' && process.env.JENKINS_HOME !== undefined);
```

But I can push another commit for that, np.